quick-install/trow-gke.yaml

```diff
@@ -100,6 +100,22 @@ spec:
           name: cert-vol
         - mountPath: /data
           name: data-vol
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8443
+            scheme: HTTPS
+            httpHeaders:
```

Comment:
What are the headers for?

Reply:
I added them incase you would need to tell apart livelivess calls